xml/System.IO.Pipes/NamedPipeServerStreamAcl.xml

```diff
@@ -45,19 +45,51 @@
         <Parameter Name="additionalAccessRights" Type="System.IO.Pipes.PipeAccessRights" />
       </Parameters>
       <Docs>
-        <param name="pipeName">To be added.</param>
-        <param name="direction">To be added.</param>
-        <param name="maxNumberOfServerInstances">To be added.</param>
-        <param name="transmissionMode">To be added.</param>
-        <param name="options">To be added.</param>
-        <param name="inBufferSize">To be added.</param>
-        <param name="outBufferSize">To be added.</param>
-        <param name="pipeSecurity">To be added.</param>
-        <param name="inheritability">To be added.</param>
-        <param name="additionalAccessRights">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="pipeName">The name of the pipe.</param>
+        <param name="direction">One of the enumeration values that determines the direction of the pipe.</param>
+        <param name="maxNumberOfServerInstances">The maximum number of server instances that share the same name. You can pass <see cref="F:System.IO.Pipes.NamedPipeServerStream.MaxAllowedServerInstances" /> for this value.</param>
+        <param name="transmissionMode">One of the enumeration values that determines the transmission mode of the pipe.</param>
+        <param name="options">One of the enumeration values that determines how to open or create the pipe.</param>
+        <param name="inBufferSize">The input buffer size.</param>
+        <param name="outBufferSize">The output buffer size.</param>
+        <param name="pipeSecurity">An object that determines the access control and audit security for the pipe.</param>
+        <param name="inheritability">One of the enumeration values that determines whether the underlying handle can be inherited by child processes.</param>
+        <param name="additionalAccessRights">One of the enumeration values that specifies the access rights of the pipe.</param>
+        <summary>Creates a new instance of the <see cref="T:System.IO.Pipes.NamedPipeServerStream" /> class with the specified pipe name, pipe direction, maximum number of server instances, transmission mode, pipe options, recommended in and out buffer sizes, pipe security, inheritability mode, and pipe access rights.</summary>
+        <returns>A new named pipe server stream instance.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If `options` contains &lt;xref:System.IO.Pipes.PipeOptions.CurrentUserOnly&gt;, the passed `pipeSecurity` is ignored and the returned &lt;xref:System.IO.Pipes.NamedPipeServerStream&gt; object is created using a custom &lt;xref:System.IO.Pipes.PipeSecurity&gt; instance assigned to the current Windows user as its only owner with full control of the pipe.
```

Comment:
```suggestion
If `options` contains <xref:System.IO.Pipes.PipeOptions.CurrentUserOnly>, the passed `pipeSecurity` is ignored and the returned <xref:System.IO.Pipes.NamedPipeServerStream> object is created using a custom <xref:System.IO.Pipes.PipeSecurity> instance assigned to the current Windows user as its only owner with full control of the pipe.
```

Reply:
That's a bug in the tool, I should correct it.